builtin/mainmenu/tab_online.lua

```diff
@@ -362,6 +369,12 @@ local function main_button_handler(tabview, fields, name, tabdata)
 		return true
 	end
 
+	if fields.btn_server_url then
+		local url = find_selected_server().url
```

Comment:
could inline the `url` variable

Reply:
Yep, I even had it like that before. Will revert :+1: